README.md

```diff
@@ -58,6 +71,52 @@ Example:
   ]
 ```
 
+Alternatively enable private endpoints:
+
+```terraform
+locals {
+  private_endpoint_rg_name   = var.businessArea == "sds" ? "ss-${var.env}-network-rg" : "${var.businessArea}-${var.env}-network-rg"
+  private_endpoint_vnet_name = var.businessArea == "sds" ? "ss-${var.env}-vnet" : "${var.businessArea}-${var.env}-vnet"
+}
+
+# CFT only, on SDS remove this provider
+provider "azurerm" {
+  alias           = "private_endpoints"
+  subscription_id = var.aks_subscription_id
+  features {}
+  skip_provider_registration = true
+}
+
+data "azurerm_subnet" "private_endpoints" {
```

Comment:
could add to module directly like in servicebus and postgres flexible server but it would be a breaking change so just enabled the capability in this